docs/references/backend/sessions/authenticate-request.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: authneticateRequest() | User Object
+description: Networkless method to verify a token passed from the frontend.
+---
+
+# `authenicateRequest()`
+
+Networkless method to authenicate a token passed from the frontend..
+
+```tsx
+const authStatus = await clerkClient.authenticateRequest();
+```
+
+## `AuthenticateRequestParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `request` | `IncomingMessage` | `req` object |
+| `secretKey?` | `string` | Secret Key (same as apiKey) |
+| `publishableKey?` | `string` | Clerk Publishable Key |
+| `domain?` | `string` | Domain for the application |
```

Comment:
I think this is also part of multi-domain setup.
cc: @panteliselef

Reply:
@panteliselef Just looking for feedback on the above. Thanks.

Reply:
Yes, this is the part of the multidomain feature. But the description seems correct. In this field we need to pass the domain that the satellite application will be hosted at.

if my satellite app is accessible from https://satellite.dev, i need to pass satellite.dev as the value. For dev and localhost you can pass f.e. localhost:3000